demes/demes.py

```diff
@@ -1326,8 +1334,7 @@ def deme(
         end_time=None,
         initial_size=None,
         final_size=None,
```

Comment:
It looks like you've removed the calls to this function that use the start_time, end_time, initial_size, and final_size params. Was it your intention to remove the args here too?